README.md

```diff
@@ -200,3 +173,13 @@ Below are a few helpful aliases that have been suggested:
     # for the pull request.
     # git qa-pr maas-docs 6 qa-sticky-headers
     qa-pr = "!sh -c 'git checkout develop; git pull $0 develop; git checkout -b $2; git fetch-pr $0 $1; git merge pr/$1'"
+
+[contributing]: http://maas.io/docs/contributing
```

Comment:
I think we have to - I just tested that link with the html added and it doesn't work.